lib/shopify_app/controller_concerns/login_protection.rb

```diff
@@ -170,7 +170,9 @@ def fullpage_redirect_to(url)
     end
 
     def current_shopify_domain
-      shopify_domain = sanitized_shop_name || session[:shopify_domain]
+      shopify_domain = sanitized_shop_name ||
+        session[:shopify_domain] ||
+        jwt_shopify_domain
```

Comment:
Minor: can we check for `jwt_shopify_domain` before `session[:shopify_domain]` since we're doing the same approach for when getting the sessions?
```
shopify_domain = sanitized_shop_name ||
        jwt_shopify_domain ||
        session[:shopify_domain]